addOns/scripts/CHANGELOG.md

```diff
@@ -4,7 +4,11 @@ All notable changes to this add-on will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
+### Added
+- Support for loadDir action, which loads all of the scripts under the specified directory..
```

Comment:
- Double dot.
- I think we should be more explicit this is for the AF job (same for the other entry).